helper-install-puppeteer-deps

```diff
@@ -19,3 +20,6 @@ sudo apt-get install -yq \
     libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 \
     libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates \
     fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
```

Comment:
Why not just add `jq` on this line?

```suggestion
    fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget jq
```

Reply:
Ah, so it felt wrong enough to add `jq` in this file already as it isn't a puppeteer dependency.

I've separated it and have the comment to make that clear.